client/src/app/routes/stardust/AddressPage.tsx

```diff
@@ -95,8 +98,10 @@ const AddressPage: React.FC<RouteComponentProps<AddressRouteProps>> = (
 
     useEffect(() => {
         if (bech32AddressDetails) {
-            // eslint-disable-next-line no-void
-            void loadAddressData();
+            void getAddressBalance();
+            void loadBasicAddressData();
+            void loadAliasAddressData();
+            void loadNftAddressData();
```

Comment:
Naming for these methods above sounds a bit misleading, just call it what it is... `loadAddressBasicOutputs`, `loadAddressAliasOutputs`